doc/ngsi/docs/JSONParser.md

```diff
@@ -41,19 +42,19 @@ Output
 }
 ```
 #### Attribute with metadata 
-When you want to convert an attribute with metadata to a NGSI attribute, you can use the `parseAttrs()` function as the example above. For example, in the next fragment of code, the `parseAttrs()` function receives as parameter a JSON with the following data: the attribute `temperature` with value of `50` and as metadata of this attribute, the attributes `frequency` with value of `50`, and the attribute `scale` with value `'Fahrenheit'`.
+When you want to convert an attribute with metadata to a NGSI attribute, you can use the `parseAttrs()` function as the example above, but in the definition of the attribute you must include the metadata. For example, in the next fragment of code, the `parseAttrs()` function receives as parameter a JSON with the following: the attribute `temperature` with value of `50`; and as metadata of this attribute, the attribute `frequency` with value of `50`, and the attribute `scale` with value `Fahrenheit`.
```

Comment:
I know this is an example, so nothing serious, but is NGSI using `scale` rather than `unit`?